apps/token-manager/package.json

```diff
@@ -37,22 +35,23 @@
   ],
   "license": "(GPL-3.0-or-later OR AGPL-3.0-or-later)",
   "description": "",
-  "devDependencies": {
-    "@aragon/apps-shared-migrations": "1.0.0",
-    "@aragon/apps-shared-scripts": "^1.0.0",
-    "@aragon/cli": "^6.0.0",
-    "@aragon/test-helpers": "^2.1.0",
-    "@aragon/truffle-config-v4": "^1.0.1",
-    "eth-gas-reporter": "^0.2.0",
-    "ethereumjs-testrpc-sc": "^6.5.1-sc.0",
-    "ganache-cli": "^6.4.3",
-    "solidity-coverage": "0.6.2",
-    "solium": "^1.2.3",
-    "truffle": "4.1.14",
-    "truffle-extract": "^1.2.1"
-  },
   "dependencies": {
-    "@aragon/apps-shared-minime": "1.0.0",
+    "@aragon/minime": "1.0.0",
     "@aragon/os": "4.2.0"
+  },
+  "devDependencies": {
+    "@aragon/buidler-aragon": "^0.2.2",
+    "@aragon/contract-test-helpers": "0.0.1",
+    "@aragon/minime": "1.0.0",
+    "@nomiclabs/buidler": "^1.0.2",
+    "@nomiclabs/buidler-etherscan": "^1.2.0",
+    "@nomiclabs/buidler-solhint": "^1.2.0",
+    "@nomiclabs/buidler-truffle5": "^1.1.2",
+    "@nomiclabs/buidler-web3": "^1.1.2",
+    "bignumber.js": "^9.0.0",
+    "buidler-gas-reporter": "^0.1.3",
+    "eth-ens-namehash": "^2.0.8",
```

Comment:
I'll move this one to contracts-helpers.

Reply:
Is this only used by contract-helpers internal logic?

Reply:
No, it's used in some tests on a couple of apps.